h/templates/accounts/developer.html.jinja2

```diff
@@ -5,25 +5,55 @@
 
 {% block page_content %}
   <div class="form-vertical">
+    {% if not feature('activity_pages') -%}
     <h2 class="form-heading">
       <span>{% trans %}Generate your API token{% endtrans %}</span>
     </h2>
+    {% endif %}
 
     {% if token %}
+      {% if feature('activity_pages') %}
+        <p class="form-help-text">
+        <label for="token">{% trans %}Your API token is:{% endtrans %}</label>
+        </p>
+        <input id="token"
+               type="text"
+               value="{{ token }}"
+               readonly="readonly"
+               class="api-token">
+        <p class="help-block form-help-text">
```

Comment:
`grep` tells me that `help-block` is defined in three templates but not actually styled anywhere (except in `front-page.css` which is not used in any of the three pages that reference this class), so we can remove it.